src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.cs

```diff
@@ -15,39 +15,39 @@ namespace System.Text
     public static partial class PrimitiveFormatter
     {
         #region Integers
-        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
```

Comment:
How did you chose between the two alternative orderings we could use? i.e. why do you think format followed by encoding is better than the other way around?